### PR TITLE
feat: add skipRegex

### PR DESCRIPTION
**Pull request by jitsedesmet**
This PR adds the possibility to skip tests. This feature will allow more control to the users.
I decided to add a skipRegex that works similar to the already existing `-t` test filtering option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,18 @@ $ rdf-test-suite myengine.js http://w3c.github.io/rdf-tests/sparql/sparql11/mani
...
+### Test skipping
+
+With the `--skip` option you can filter tests that should be skipped
+based on a regex that will be matched with the test URI.
+
+For example:
+
+```bash
+$ rdf-test-suite myengine.js http://w3c.github.io/rdf-tests/sparql/sparql11/manifest-all.ttl \
+  --skip test_pn_bad_0
```
Comment (rubensworks) [inline README.md:117]: Could we add an example of a case where we need to skip 2 tests?
Just so that people can just copy-paste that format in such cases.

Comment (jitsedesmet) [reply README.md:117]: Would you like it to be with the `(a|b)` notation or the `a.*`. Probably `a|b`?

Comment (rubensworks) [reply README.md:117]: `(a|b)` might be better, as it's more easily extensible.

**Review by rubensworks — APPROVED** [review]
Thanks!

**Comment by rubensworks** [comment]
Released as 1.24.0.

**Pull request merged**